modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/transaction/TransactionBindingDelegate.java

```diff
@@ -123,6 +124,11 @@ public void invalidate() {
                                                                   extensionModel.getName(),
                                                                   connection.getClass().getName())));
       }
+    } catch (Exception e) {
```

Comment:
this changes looks good and I understand there's a functional test in the jms connector, at the very least a unit test should exist over this